templates/netdata.yml.j2

```diff
@@ -1,3 +1,7 @@
+{% for host in groups[prometheus_netdata_group] %}
 - labels:
     module: netdata
-  targets: {{ prometheus_netdata_targets }}
+    host: "{{ hostvars[host].ansible_nodename }}"
+  targets:
+    - {{ hostvars[host]['ansible_' + prometheus_netdata_bind_interface].ipv4.address + ':' + prometheus_netdata_port|string }}
```

Comment:
Same as in the defaults; this needs to be able to handle openio_bind_mgmt_interface and openio_bind_mgmt_address.

Reply:
@vdombrovski updated.